httpserver/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 2.8.0)
 
 project(httpserver)
 
+add_definitions(-DBOOST_LOG_DYN_LINK)
```

Comment:
I'm afraid it's more complicated than this, as it depends on how your boost installation was built.
Here's from [boost docs](https://www.boost.org/doc/libs/1_69_0/libs/log/doc/html/log/rationale/namespace_mangling.html)
> Namespace mangling may lead to linking errors if the application is misconfigured. One common mistake is to build dynamic version of the library and not define BOOST_LOG_DYN_LINK or BOOST_ALL_DYN_LINK when building the application, so that the library assumes static linking by default.

We can't assume static or dynamic linking in our cmake, so we might defer that kind of flags as a command-line argument when calling cmake.
e.g. `cmake -DBOOST_LOG_DYN_LINK=ON ../httpserver` should work.

Another option would be to lock-in our cmake to only use static libraries, in order to make our binaries more portable, but heavier (they contain all their necessary symbols, no dynamic linking needed). Not sure what's best at this stage for the storage server 🤔